src/languages/es.ts

```diff
@@ -5197,7 +5197,8 @@ const translations = {
                 delegateSubmit: ({delegateUser, originalManager}: DelegateSubmitParams) => `envié este informe a ${delegateUser} ya que ${originalManager} está de vacaciones`,
                 exportedToCSV: `exportó este informe a CSV`,
                 exportedToIntegration: {
-                    automatic: ({label}: ExportedToIntegrationParams) => `exportó este informe a ${label}.`,
+                    automaticOne: ({label}: ExportedToIntegrationParams) => `Exportó automáticamente este informe a ${label} a través de la `,
```

Comment:
- Lowercase, since it's an automated message, right?
- Also, do we need the trailing space here? I don't see it in the English copy
```suggestion
                    automaticOne: ({label}: ExportedToIntegrationParams) => `exportó automáticamente este informe a ${label} a través de la `,
```

Reply:
> * Also, do we need the trailing space here? I don't see it in the English copy

Bump on this question